### PR TITLE
Fix pyup config

### PR DESCRIPTION
**Pull request by stephendonner**
@davehunt @tarekziade r?

### CODE REVIEW

File: .pyup.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# autogenerated pyup.io config file 
+# autogenerated pyup.io config file
 # see https://pyup.io/docs/configuration/ for all available options
 
 update: insecure
+requirements:
+  - Pipfile
+  - pipenv.txt
```
Comment (davehunt) [inline .pyup.yml]: There is no pipenv.txt

File: .pyup.yml [hunk 2]
```diff
@@ -1,4 +1,7 @@
-# autogenerated pyup.io config file 
+# autogenerated pyup.io config file
 # see https://pyup.io/docs/configuration/ for all available options
```
Comment (davehunt) [inline .pyup.yml]: We can remove this comment, as it's no longer the autogenerated file.

File: .pyup.yml [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
 
 update: insecure
+requirements:
+  - Pipfile
```
Comment (davehunt) [inline .pyup.yml]: We should consider adding Pipfile.lock, as that's currently used for the installation with Servicebook.

File: .pyup.yml (hunk 1 above)
Comment (stephendonner) [reply .pyup.yml]: Sorry, yeah, there is/will be, over in https://github.com/mozilla/servicebook/pull/61 - I'll remove this here, as I was intending to make these changes clean and separate.

Comment (stephendonner) [reply .pyup.yml]: Fixed.

File: .pyup.yml (hunk 2 above)
Comment (stephendonner) [reply .pyup.yml]: Done.

File: .pyup.yml (hunk 3 above)
Comment (stephendonner) [reply .pyup.yml]: Added!

**Comment by stephendonner** [comment]
@davehunt should be good to go again, now; r?  Thanks!

**Pull request merged**